packages/truffle-core/cli.js

```diff
@@ -1,20 +1,28 @@
 #!/usr/bin/env node
 require('source-map-support/register')
 
-var Config = require("truffle-config");
-var Command = require("./lib/command");
-var TaskError = require("./lib/errors/taskerror");
-var TruffleError = require("truffle-error");
+const TaskError = require("./lib/errors/taskerror");
+const TruffleError = require("truffle-error");
 
-var command = new Command(require("./lib/commands"));
+const nodeMajorVersion = parseInt(process.version.slice(1));
```

Comment:
Not to say this is worth changing, but FYI the "fancy" way to do this might be to use the [semver](https://www.npmjs.com/package/semver) NPM package to get the version number.

Actually you might want to change to this because Node `"v10.x.x"` will slice incorrectly.

Reply:
Why won't it slice correctly?  The slice should just remove the leading 'v'.  Will the version number not always be prefaced by 'n'?

Reply:
```
> parseInt('v7.0.0'.slice(1)) < 8
true
> parseInt('v10.13.0'.slice(1)) < 8
false
```

Hm. Seems fine. JS you slay me